cmd/crypto/sse-kms.go

```diff
@@ -109,7 +114,7 @@ func (s3 ssekms) UnsealObjectKey(kms KMS, metadata map[string]string, bucket, ob
 // the modified metadata. If the keyID and the kmsKey is not empty it encodes
 // both into the metadata as well. It allocates a new metadata map if metadata
 // is nil.
-func (ssekms) CreateMetadata(metadata map[string]string, keyID string, kmsKey []byte, sealedKey SealedKey) map[string]string {
+func (ssekms) CreateMetadata(metadata map[string]string, keyID string, kmsKey []byte, sealedKey SealedKey, ctx Context) map[string]string {
```

Comment:
```suggestion
func (ssekms) CreateMetadata(metadata map[string]string, keyID string, kmsKey []byte, sealedKey SealedKey, ctx Context) map[string]string {
```

Taking context at the end wouldn't this confuse?

Reply:
This is not the `context.Context` from the standard library. This is the KMS context that contains data that should be cryptographically bound to the object.

Reply:
This one: https://github.com/minio/minio/blob/master/pkg/kms/context.go#L33